### PR TITLE
Enabled evaluate number as int in query

### LINKED ISSUE #63: Feature Request for `base search --query`
# Motivation
When I try `base search mnist --query "id  <=  1200"` command,
now, they are evaluated in lexical order as str types, not int types.
So, for example, data with id=10000 will also be obtained in this case.

### PR DESCRIPTION
**Pull request by 31159piko-suke**
close #63 

- Also fixed a bug in project.py

### CODE REVIEW

**Comment by 31159piko-suke** [comment]
With this modification, both of the command that `base search mnist --query "id <= 7"` and `base search --query "id <= '7'"` evaluate number as int.

So we need an option to evaluate the numbers as strings in lexical order, but I do not think this is a priority...so it is not implemented at this time.

File: base/files.py [hunk 1]
```diff
@@ -256,26 +256,48 @@ def __query_filter(self, result: List[dict], query: List[str] = []) -> List[dict
...
+                # if q = "label <= 7" or  q = "label <= '7'"
+                # key = "label", value = "7", operator = "<="
                 query_split = q.split()
                 key = query_split[0]
-                value = query_split[-1]
+                value = query_split[-1].lstrip("'").rstrip("'").lstrip('"').rstrip('"')
                 operator = " ".join(query_split[1:-1])
 
-                if operator in ["==", ">", ">=", "<", "<="]:
+                if operator in "==":
```
Comment (kenichihiguchi) [inline base/files.py]: You should write as `if operator == "==":`, because `str1 in str2` evaluates str2 as includes str1. (Ex. "=" will be accepted...)

**Review by kenichihiguchi — COMMENTED** [review]
@31159piko-suke 
thanks!
I found a bug and commented it, so please fix files.py

**Review by kenichihiguchi — APPROVED** [review]
I fixed it

**Comment by 31159piko-suke** [comment]
thanks!

**Pull request merged**